docs/upgrade-1.x-to-2.0.md

```diff
@@ -0,0 +1,76 @@
+# Guide to upgrade daobet node from 1.x to 2.0
+
+Please, read this guide carefully before performing any actions.
+
+Here, `data_dir` variable is a path to you `data` directory (contains `blocks`, `state`, and other files and
+directories).
+
+* Stop running node (binary or container):
+  ```
+  kill daobet-node
+  # OR
+  docker stop daobet-producer
+  ```
+
+* Backup `blocks/reversible`, `state-history` and `state` directories located in node data directory:
+  ```
+  ( cd "$data_dir" && rsync -avR --delete blocks/reversible state-history state ~/backup.daobet-1.x )
+  ```
+
+* If you used default values for `data-dir` and/or `config-dir`, you may need to rename `nodeos` directory to the
+  `daobet-node`.
+
+* Delete the following directories within the node data directory: `blocks/reversible`, `state-history`, and `state` (be
+  sure they were backed up earlier):
+  ```
+  rm -rf /path/to/daobet-node/data/{blocks/reversible,state-history,state}/
+  ```
+
+* Create file `protocol_features/BUILTIN-PREACTIVATE_FEATURE.json` in your config directory with the following contents:
+  ```
+  {
+    "protocol_feature_type": "builtin",
+    "dependencies": [],
+    "description_digest": "64fe7df32e9b86be2b296b3f81dfd527f84e82b98e363bc97e40bc7a83733310",
+    "subjective_restrictions": {
+      "earliest_allowed_activation_time": "1970-01-01T00:00:00.000",
```

Comment:
mb set real time ? like `2020-04-23`, it's just example, need to discuss about particular date ?

Reply:
@algys, this date should be chosen on BPs' quorum. Date in the file is just an example for understanding format of the value.

Reply:
Ok, but I guess we should add text about it(that before updating BPs need to choose that date) to instruction

Reply:
See line 44.